airwaveapiclient/airwaveapiclient.py

```diff
@@ -257,7 +257,35 @@ def client_detail(self, mac):
         params = {'mac': mac}
         params = AirWaveAPIClient.urlencode(params)
         return self.session.get(url, verify=False, params=params)
+    
+    def client_location(self, mac):
+        """Client detail information.
+
+        Args:
+
+            :mac (str): Client device's MAC address.
+
+        Returns:
+
+            :Response: requests.models.Response.
 
+        Usage: ::
+
+            >>> res = airwave.client_detail('12:34:56:78:90:AB')
```

Comment:
ah i haven't update the samples usage: :: res = airwave.client_location('12:34:56:78:90:AB')